container-desktop/Configuration/ContainerDesktopConfiguration.cs

```diff
@@ -81,4 +81,27 @@ public string DaemonConfig
     public ObservableCollection<CertificateInfo> Certificates { get; }
 
     public IEnumerable<CertificateInfo> GetCertificates() => CertificateInfo.GetCertificates();
+
+    [UIEditor(UIEditor.RadioList)]
+    [Display(Name = "Host Entry Mode", GroupName = ConfigurationGroups.Network, Description = "Determines how the ip address for the host entries 'host.docker.internal' and 'gateway.docker.internal' are set.", Order = 3)]
+    [Category(ConfigurationCategories.Basic)]
+    public HostEntryMode HostEntryMode
+    {
+        get => GetValue<HostEntryMode>();
+        set => SetValueAndNotify(value);
+    }
+
+    [Display(Name = "Host Entry Adapter", GroupName = ConfigurationGroups.Network, Order = 4)]

```

Comment:
This list is not updated automatically, either refresh or add an description to restart if an interface is enabled or connected (up) after startup.